src/components/hoc/status.js

```diff
@@ -0,0 +1,34 @@
+import { update } from 'ramda';
+import { compose, withState, withHandlers } from 'recompose';
```

Comment:
But we have webpack 2 tree shaking too, that would do the same thing, only bundle the imported stuff.